core/src/main/resources/hudson/model/Label/index.jelly

```diff
@@ -38,19 +38,40 @@ THE SOFTWARE.
       <j:forEach var="a" items="${it.allActions}">
         <st:include page="summary.jelly" from="${a}" optional="true" it="${a}" />
       </j:forEach>
-
       <div>
         <h2>${%Nodes}</h2>
-        <j:forEach var="n" items="${it.sortedNodes}">
-          <j:set var="c" value="${app.getComputer(n.nodeName)}"/>
-          <j:set var="url" value="${rootURL}/${c.url}"/>
-          <nobr>
-            <a href="${url}" class="jenkins-link--with-icon"><l:icon src="${c.iconClassName}" /></a>
-            <st:nbsp/>
-            <a href="${url}" class="model-link inside">${c.displayName}</a>
-          </nobr>
-          <wbr/>
-        </j:forEach>
+        <table id="nodes_${it.displayName}" class="jenkins-table sortable">
+            <thead>
+              <tr>
+                <th class="jenkins-table__cell--tight">S</th>
+                <th initialSortDir="down">${%Name}</th>
+                <th>${%Description}</th>
+                <th align="right" tooltip="Number of total executors that belong to this node">${%Configured executors}</th>
+                <th align="right" tooltip="Number of busy executors that are carrying out some work right now">${%Busy executors}</th>
+                <th align="right" tooltip="Number of idle executors that can start working immediately">${%Idle executors}</th>
+              </tr>
+            </thead>
+            <tbody>
+            <j:forEach var="n" items="${it.sortedNodes}">
+              <j:set var="c" value="${app.getComputer(n.nodeName)}"/>
+              <j:set var="url" value="${rootURL}/${c.url}"/>
+                <tr id="node_${c.name}">
```

Comment:
Could this result in illegal keys?

Reply:
what you mean with illegal key? Or better what are legal key. Just inspired by tables in /computer or in /label/*. I have no idea why is there key used? I think, that it is some old html style in jenkins.

Reply:
Interesting,

> Technically, in [HTML5](https://developer.mozilla.org/en-US/docs/Glossary/HTML5), the value for an id attribute may contain any character, except [whitespace](https://developer.mozilla.org/en-US/docs/Glossary/Whitespace) characters.

So only whitespace would be illegal, and https://github.com/jenkinsci/jenkins/blob/39fd38fd86d7e7cb0a13e32562d36d67ff52a5f9/core/src/main/java/hudson/model/Computer.java#L1490 prevents that.

Reply:
Right, that other location is probably not working well either.

I suggest you remove it if it's unused anyway, or if you don't feel like doing that it's probably fine to keep.

Reply:
I am not sure, but this looks like dead code:
https://github.com/jenkinsci/jenkins/blob/39fd38fd86d7e7cb0a13e32562d36d67ff52a5f9/core/src/main/resources/hudson/model/Label/index.jelly#L38

know some body what shall do that?

Reply:
It's not dead code, it's part of Jenkins's extensibility model. `Label` is `Actionable`: https://github.com/jenkinsci/jenkins/blob/39fd38fd86d7e7cb0a13e32562d36d67ff52a5f9/core/src/main/java/hudson/model/Label.java#L86

See https://www.jenkins.io/doc/developer/extensibility/#actionable-and-actions for a (very short) introduction and references.

Reply:
Works well also without ID. Tested also with node that contains space in name.